python/ml4ir/build/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Using python based centos7 docker base image:
-FROM ducouloa/ml4ir:0.1
+FROM lastmansleeping/ml4ir:0.1
```

Comment:
I had to update and re-release the base image for a java 11 upgrade recently.

Reference -> https://github.com/salesforce/ml4ir/pull/197